lib/course_planner/accounts/user.ex

```diff
@@ -45,6 +45,12 @@ defmodule CoursePlanner.Accounts.User do
       greater_than_or_equal_to: 1, less_than_or_equal_to: 7)
   end
 
+  def changeset(model, params, :bulk) do
+    model
+    |> changeset(params)
+    |> validate_required([:role])
```

Comment:
Why is `role` the only required field in here? Shouldn't `email` be here as well?

Reply:
because email is already is a required field in "changeset(params)" but not role

Reply:
Sorry, I forgot my eyes somewhere :D

Reply:
🤔  role should probably be required in the default changeset